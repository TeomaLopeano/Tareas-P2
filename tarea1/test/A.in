cantidadEnCadena
estaEnCadena 5
insertarAlInicio 5 5.0
infoCadena 5
estaEnCadena 3
imprimirCadena
cantidadEnCadena
cantidadColCadenas 1
insertarEnColCadenas 3 3.0 1
cantidadColCadenas 1
estaEnColCadenas 3 1
estaEnColCadenas 5 1
estaEnColCadenas 3 8
infoEnColCadenas 3 1
Fin