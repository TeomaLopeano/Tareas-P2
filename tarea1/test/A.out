1>La cantidad es 0.
2>5 no está en la cadena.
3>Se insertó (5,5.00) al inicio de la cadena.
4>El elemento es (5,5.00)
5>3 no está en la cadena.
6>(5,5.00)
7>La cantidad es 1.
8>La cantidad en la cadena 1 es 0.
9>Se insertó (3,3.00) en la cadena 1.
10>La cantidad en la cadena 1 es 1.
11>3 está en la cadena 1.
12>5 no está en la cadena 1.
13>3 no está en la cadena 8.
14>La primera aparición de 3 en la cadena 1 es (3,3.00)
15>Fin.
